src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -50,6 +50,17 @@ trait Backend {
   def initializeForWorkflow(workflow: WorkflowDescriptor): HostInputs
 
   /**
+   * Return the implementation of all WDL functions (e.g. read_lines(), stdout()) used
+   * by this backend
+   */
+  def engineFunctions(context: TaskExecutionContext): WdlStandardLibraryFunctions
+
+  /**
+   *
```

Comment:
To make life easier on an upcoming tech debt ticket, can you comment this, what it is intended to do and what everything is? The goal would be so that someone implementing a new backend would know what this is and what to do w/ it. Also the same for `engineFunctions` but that description might be enough here.